zb-db/src/test/java/io/camunda/zeebe/db/impl/rocksdb/transaction/ForeignKeyCheckerTest.java

```diff
@@ -58,6 +62,33 @@ void shouldSucceedOnExistingForeignKey() throws Exception {
     check.assertExists(tx, new DbForeignKey<>(key, TestColumnFamilies.TEST_COLUMN_FAMILY));
   }
 
+  @Test
+  void shouldSucceedOnRealColumnFamily(@TempDir final File tempDir) throws Exception {
+    // given
+    final var db = DefaultZeebeDbFactory.<TestColumnFamilies>getDefaultFactory().createDb(tempDir);
+    final var txContext = db.createContext();
+
+    final var cf1 =
+        db.createColumnFamily(
+            TestColumnFamilies.TEST_COLUMN_FAMILY, txContext, new DbLong(), DbNil.INSTANCE);
+
+    final var check =
+        new ForeignKeyChecker(
+            (ZeebeTransactionDb<?>) db, new ConsistencyChecksSettings(true, true));
+
+    // when -- key 1 exists in first column family
+    final var cf1Key = new DbLong();
+    cf1Key.wrapLong(1);
+    cf1.insert(cf1Key, DbNil.INSTANCE);
+
+    // then -- referring to key 1 does not throw
+    check.assertExists(
```

Comment:
:wrench: there is something like `Assertions.assertNoExceptionThrownBy( lambda)`

Reply:
Oh my, I've been looking for this forever but somehow couldn't find. Thanks :+1:

Reply:
It's `assertDoesNotThrow` btw.